_static/js/custom.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    window.onload = function () {
+        var isApiPresent = document.getElementsByTagName('redoc').length;
+        if(isApiPresent) {
+            window.addEventListener("hashchange", function(){
+                var locationHash = location.hash.split('#')[1],
```

Comment:
I'm a bit worried what will happen if `location.hash` will contain extra `#` character, but I'm not sure if it's possible to generate such link using `redoc`.

Reply:
It's not possible.